test.ts

```diff
@@ -32,7 +36,7 @@ test.after('cleanup', async () => {
 
 const error = 'Simple error';
 
-const makeRequest = (url = 'https://httpbin.org/anything') => {
+const makeRequest = (url = 'https://httpbin.org/anything'): {request: ClientRequest; timings: timer.Timings} => {
```

Comment:
```suggestion
const makeRequest = (url = 'https://httpbin.org/anything'): {request: ClientRequest; timings: Timings} => {
```

(You need to import it, obviously)